static/src/stylesheets/module/_adslot.scss

```diff
@@ -97,6 +97,53 @@
     }
 }
 
+.ad-slot-container:has(div.ad-slot--inline),
```

Comment:
Love to see modern CSS, but I'm not sure it's safe to use :has yet given it is not yet available in firefox: https://developer.mozilla.org/en-US/docs/Web/CSS/:has#browser_compatibility

Reply:
Ah great catch, thanks Dom! I'll replicate this logic in a different way